website/docusaurus.config.ts

```diff
@@ -136,6 +136,18 @@ const config: Config = {
       },
     ],
   ],
+  customFields: {
+    glossaryItems: [
```

Comment:
You could copy how guides a set up in that way a "category" page should be automatically generated.